client/js/components/procedure/admin/DpAddOrganisationList.vue

```diff
@@ -178,6 +179,7 @@ export default {
       return Object.values(this.invitableToebItems).reduce((acc, item) => {
         const locationContactId = item.relationships.locationContacts.data[0].id
         const locationContact = this.getLocationContactById(locationContactId)
+        const hasNoEmail = item.attributes.participationFeedbackEmailAddress === null
```

Comment:
Could this also be empty string or undefined?

```suggestion
        const hasNoEmail = !item.attributes.participationFeedbackEmailAddress
```